drivers/interrupt_controller/loapic_intr.c

```diff
@@ -93,8 +94,24 @@ static int loapic_init(struct device *unused)
 	ARG_UNUSED(unused);
 	s32_t loApicMaxLvt; /* local APIC Max LVT */
 
-	/* enable the Local APIC */
-	x86_write_loapic(LOAPIC_SVR, x86_read_loapic(LOAPIC_SVR) | LOAPIC_ENABLE);
+	/*
+	 * enable the local APIC. note that we use xAPIC mode here, since
+	 * x2APIC access is not enabled until the next step (if at all).
+	 */
+
+	x86_write_xapic(LOAPIC_SVR, x86_read_xapic(LOAPIC_SVR) | LOAPIC_ENABLE);
+
+#ifdef CONFIG_X2APIC
+	/*
+	 * turn on x2APIC mode. we trust the config option, so
+	 * we don't check CPUID to see if x2APIC is supported.
```

Comment:
indent

Reply:
@nashif Not quite sure what you're looking for here.. line 102 was exactly 80 characters, which is a bit ugly, so I've broken that up.  Is that what you meant?

Reply:
ignore, my browser is showing wrong indentation.